lemur/static/js/admin_create_and_manage_lab.js

```diff
@@ -0,0 +1,78 @@
+
+$(document).ready(function(){
+    //Modify an existing lab
+    $('.modifyLab').click(function(){
+        var labId = $(this).closest('tr').data('labid');
+        window.location.replace('/admin_modify_lab/'+labId);        
+    });
+    // The reason that we use javascript to show the modal rather than html is that this enables us
+    // to show the modal that is closest to the delete button such that the modal is within the tr
+    // that we want to delete
+    $('.deleteLab').click(function(){
+        $(this).closest('tr').find('#deleteConfirm').modal("show");
+    });
+
+    //Duplicate/Delete a lab
+    $('.duplicateLab').add($('button[name=confirm]')).click(function(){
+        var operation = $(this).data('operation');
+        var labId = $(this).closest('tr').data('labid');
+        //Communicate the name of the lab to be duplicated with python via Ajax 
+        $.ajax({
+          type: 'POST',
+          contentType: 'application/json',
+          dataType: 'json',
+          url: 'http://127.0.0.1:5000/_admin_'+operation,
+          data: JSON.stringify({'labId':labId}),
+          success: function(result){
+                    console.log(operation+' successfully');               
+                },
+          error : function(result){
+                  console.log('Fail to '+operation);
```

Comment:
you probably want to tell the user that this happened (it's cool if you just make a ticket and do it laterz)